test_oidc.py

```diff
@@ -171,6 +176,16 @@ def test_bad_logout_uri(login_user, resource_url,
         4. Success
         5. Page should be without redirects
     """
+    LOGGER.info("Setting OIDCRedirectURLsAllowed option in config")
+    conf_path = "/etc/httpd/conf.d/openidc_example_app_oidc_keycloak_master.conf"
+    backup = ["cp", conf_path, f"{conf_path}.test_backup"]
+    subprocess.run(backup)
```

Comment:
This is very non-pythonic, there are multiple ways to copy files. one well readable is to use `shutil`

`shutil.copy(conf_path, f"{conf_path}.test_backup")`

Is there any reason for using subprocess here?